source/MRMesh/MRCylinderApproximator.h

```diff
@@ -0,0 +1,413 @@
+#include "MRMeshFwd.h"
+#include "MRLine.h"
+#include "MRVector.h"
+#include "MRMatrix.h"
+#include <Eigen/Eigenvalues>
+#include "MRPch/MRTBB.h"
+#include "MRToFromEigen.h"
+#include "MRConstants.h"
+
+// https://www.geometrictools.com/Documentation/LeastSquaresFitting.pdf 
+
+namespace MR
+{
+
+//using T = float;
+template <typename T>
+class Cylinder3
```

Comment:
It might be a good idea to put Cylinder3 in a separate header (with less dependencies) than Cylinder3DApproximation has

Reply:
done